src/lib.rs

```diff
@@ -89,7 +89,7 @@ pub type Bytes = u8;
 
 /// Number of components in a vector.
 #[repr(u8)]
-#[derive(Clone, Copy, Debug, PartialEq)]
+#[derive(Clone, Copy, Debug, PartialEq, Hash, Eq)]
```

Comment:
Unfortunately still needed for VectorSize and ScalarKind.

Reply:
yes, sure :)